README.md

```diff
@@ -10,6 +10,28 @@ Creates Alma-compliant XML patron profiles from information extracted from the D
 - To lint the repo: `make lint`
 - To run the app: `pipenv run patronload --help`
 
+The Data Warehouse runs on a older version of Oracle that necessitates the `thick` mode of `python-oracledb` which requires the Oracle Instant Client Library (this app was developed with version 21.9.0.0.0.)
+
+# With Docker
+Note: as of this writing, the Apple M1 Macs cannot run Oracle Instant Client, so Docker is the only option for development on those machines. 
+
+From the project folder:
+
+1. Run `make dependencies` with appropriate AWS credentials
+
+2. Run `make dist-dev` to build the container
+
+3. Run `docker run alma-patronload-dev:latest`
+
+# Without Docker
+1. Run `make dependencies` with appropriate AWS credentials 
```

Comment:
This command doesn't install instant client so isn't sufficient to run the app. I would just include the download instructions below.